packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -53,27 +53,42 @@ export function QualityBar({ invalid, valid, empty, onClick, getDataFeature, dig
 	return (
 		<RatioBar.Composition>
 			<QualityInvalidLine
-				onClick={onClick ? (e) => onClick(e, {
-					type: QualityType.INVALID
-				}) : null}
+				onClick={
+					onClick
+						? e =>
+								onClick(e, {
+									type: QualityType.INVALID,
+								})
+						: null
+				}
```

Comment:
Is there a formating issue ? It's looking weird to me

Reply:
problem of linter ?

Reply:
Forget about it, looks good now, it was the code that was not formatted well before